docs/src/userguide/language_basics.rst

```diff
@@ -44,72 +47,137 @@ the use of ‘early binding’ programming techniques.
 C variable and type definitions
 ===============================
 
-The :keyword:`cdef` statement is used to declare C variables, either local or
-module-level::
+C variables can be declared by using the :keyword:`cdef` statement, by annotating
+variable by special cython type or using function ``declare()``. Statement :keyword:`cdef`
```

Comment:
It will be better to put them into list like
- one
- two
- three

It's not easy to understand this first time reading

Reply:
done.